README.md

```diff
@@ -19,22 +19,41 @@ You can either use Donut directly or check out the available plugins:
 ## Release Notes
 See what's new [here](release-notes.md)
 
-### download
+### Download
 ```
 wget http://repo1.maven.org/maven2/io/magentys/donut/1.0/donut-1.0-one-jar.jar
 ```
 or download the latest release from: [here](http://repo1.maven.org/maven2/io/magentys/donut/1.0/donut-1.0-one-jar.jar)
 
-### run from command line
+### Run from command line
 
 ```
 java -jar donut-<Version>.jar -s cucumber:/my/path/cucumber-reports -n myProjectName
 ```
 
-### options
+### Run using Docker
+
+Donut is packaged as a [Docker container](https://hub.docker.com/r/donutreport/donut-report/)
+
+To run using Docker:
+```
+docker run -v /path/to/your/cucumber-reports:/source -v /path/to/output-report:/output donutreport/donut-report -n myProjectName -s cucumber:/source [options]
+```
+
+In the above command, using the docker `-v` parameter `/source` and `/output` volumes are mounted in the container, to pass data in, and out.
+Use `-s cucumber:/source` Donut option to retrieve the reports
+
+*Passing multiple source directories*:
+If you need to pass multiple source directories, e.g. for Cucumber reports and unit tests, and the directories are in different paths, you can use the following variant:
+
+```
+docker run -v /path/to/your/cucumber-reports:/source -v /path/to/your/unit-test-reports:/source_unit -v /path/to/output-report:/output donutreport/donut-report -n myProjectName -s cucumber:/source,/source_unit [options]
+```

```

Comment:
These commands are not working:
`Error: Unknown option -s
Error: Unknown argument 'cucumber:/source'
`